### PR TITLE
Added tests and fixed throttling

### PR DESCRIPTION
**Pull request by berenda-andrey**

### CODE REVIEW

File: winter/http/throttling.py [hunk 1]
```diff
@@ -26,6 +27,8 @@ class Throttling:
 
 
 def throttling(rate: typing.Optional[str], scope: typing.Optional[str] = None):
+    if scope is None:
```
Comment (berenda-andrey) [inline winter/http/throttling.py]: it was mistake
without it, each request with thottling has same thottling

File: tests/controllers/controller_with_exceptions.py [hunk 2]
```diff
@@ -52,3 +54,11 @@ def not_declared_but_thrown(self) -> str:
     @winter.throws(ExceptionWithoutHandler)
     def declared_but_no_handler(self) -> str:
         raise ExceptionWithoutHandler()
+
+    @winter.route_get('redirect/')
+    def redirect(self) -> str:
+        raise RedirectException(redirect_to='/root/')
```
Comment (mofr) [inline tests/controllers/controller_with_exceptions.py]: Delete this method or ControllerWithRedirectException. Seems like they do the same thing.

File: tests/test_exception_handling.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 from rest_framework.test import APIClient
 
+from winter.exceptions import WinterException
 from .controllers.controller_with_exceptions import CustomException
 from .controllers.controller_with_exceptions import ExceptionWithoutHandler
 from .entities import AuthorizedUser
 
 
 @pytest.mark.parametrize(['url_path', 'expected_status', 'expected_body'], (
+        ('redirect', HTTPStatus.FOUND, '/root/'),
```
Comment (mofr) [inline tests/test_exception_handling.py]: Remove this or `test_controller_with_redirect_exception.py`

File: winter/http/throttling.py (hunk 1 above)
Comment (mofr) [reply winter/http/throttling.py]: Can't find a test which checks it.

File: tests/controllers/controller_with_exceptions.py (hunk 2 above)
Comment (berenda-andrey) [reply tests/controllers/controller_with_exceptions.py]: Done

File: winter/http/throttling.py (hunk 1 above)
Comment (berenda-andrey) [reply winter/http/throttling.py]: Added

File: tests/test_exception_handling.py (hunk 3 above)
Comment (berenda-andrey) [reply tests/test_exception_handling.py]: Done

**Pull request merged**